lib/get-all-schemas.spec.ts

```diff
@@ -0,0 +1,115 @@
+import { test, expect, describe } from "vitest";
+import { getAllSchemas } from "./experimental.js";
+import { registerSchema } from "./schema.js";
+import "../draft-2020-12/index.js";
+import "../draft-2019-09/index.js";
+import "../draft-04/index.js";
+import "../draft-06/index.js";
+import "../draft-07/index.js";
+import "../openapi-3-0/index.js";
+import "../openapi-3-1/index.js";
+import "../stable/index.js";
```

Comment:
I don't think it's necessary to load all the dialect for this test. One is enough to test the functionality and it keeps the test cleaner and more focused on what's actually being tested.

Also, it's always best to import the same way a user would. A user can't import `schema.js` directly. That helps you test that you've exposed the function correctly. You did expose it correctly, but your test doesn't test that.

```suggestion
import { registerSchema, getAllSchemas } "../draft-07/index.js";
```